src/main/java/formflow/library/data/validators/MoneyValidator.java

```diff
@@ -22,6 +22,6 @@ public class MoneyValidator implements ConstraintValidator<Money, String> {
    */
   @Override
   public boolean isValid(String value, ConstraintValidatorContext context) {
-    return Pattern.matches("(^(0|([1-9]\\d*))?(\\.\\d{1,2})?$)?", value);
+    return value == null || Pattern.matches("(^(0|([1-9]\\d*))?(\\.\\d{1,2})?$)?", value);
```

Comment:
Hi Ana, while this is totally functional, it's a little hard to read easily. What do you think of doing this instead? 

```
  if (value != null) {
      return Pattern.matches("(^(0|([1-9]\\d*))?(\\.\\d{1,2})?$)?", value);
  }
  return true;
```

Reply:
Does this also return true if value is ""? If not, we might want to add that as well?